nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -542,7 +550,7 @@ NavfnPlanner::getCostmap(
   auto request = std::make_shared<nav2_util::CostmapServiceClient::CostmapServiceRequest>();
   request->specs.resolution = 1.0;
 
-  auto result = costmap_client_.invoke(request, 5s);
+  auto result = costmap_client_->invoke(request, 5s);
```

Comment:
Per feedback on another PR and our group discussion, we should parameterize these timeouts. This doesn't have to happen with this PR, but we should open an issue on this. Probably have a service_call_timeout parameter or something like that. We should standardize the name because it will be needed in several modules and should use the same name (not that they are necessarily the same value).